frontend/document/pdfdocument.lua

```diff
@@ -129,6 +129,10 @@ function PdfDocument:nativeToPageRectTransform(pageno, rect)
     return self.koptinterface:nativeToPageRectTransform(self, pageno, rect)
 end
 
+function PdfDocument:getSelectedWordContext(selected_text, nb_words)
+    return self.koptinterface:getSelectedWordContext(selected_text.text, selected_text.pos0, nb_words)
```

Comment:
I didn't see it previously - but it jumped at me in the credocument.
Logically/architecturally, *document shouldn't have to know and depend on the structure of `selected_text`, which is a ReaderHighlight thing.
So, all getSelectedWordContext() should have signature more similar to that koptinterface one, taking only strings as parameters.
But then, cre and pdf don't need the same parameters... so, the signature should be flexible and accomodate both usages.
May  be it was just fine the way it was, cre simple stuff in readerhighlight :/

Reply:
You didn't miss it :) I had changed the signature. Now it's changed again to accommodate both cases.

Reply:
I just thought the word is somehow more significant in the semantics of this function. Don't know...